README.md

```diff
@@ -1,32 +1,50 @@
-# API Elements JavaScript
+# API Elements
 
-This library provides an interface to [API Elements](http://apielements.org) in JavaScript.
```

Comment:
Why this line was deleted? I think the README should have an introductory ~paragraph~ sentence.

Reply:
I understand now, the original README has been moved to the sub-package. Well still the root README should have an introductory sentence on what it contains.

Reply:
Yes, I think that can come in future PRs this is part of larger task of moving Fury-* into the repository, we'll also have API Reference docs for all of them once it is done.

Reply:
Moved to https://github.com/apiaryio/api-elements.js/issues/17